src/Components/QuickGrid/Microsoft.AspNetCore.Components.QuickGrid/src/QuickGrid.razor.cs

```diff
@@ -264,7 +264,7 @@ public Task SortByColumnAsync(ColumnBase<TGridItem> column, SortDirection direct
     /// options UI that was previously displayed.
     /// </summary>
     /// <param name="column">The column whose options are to be displayed, if any are available.</param>
-    public Task ShowColumnOptionsAsync(ColumnBase<TGridItem> column)
+    public Task ShowColumnOptionsAsync(ColumnBase<TGridItem>? column)
     {
         _displayOptionsForColumn = column;
         _checkColumnOptionsPosition = true; // Triggers a call to JS to position the options element, apply autofocus, and any other setup
```

Comment:
If called with `null`, this is still going to set `_checkColumnOptionsPosition` to true, resulting in unnecessary interop.

I'd suggest just exposing `CloseColumnOptions` instead